.github/workflows/lint-and-test.yml

```diff
@@ -29,3 +29,17 @@ jobs:
 
       - name: Test
         run: npm run test
+
+  end-to-end-test:
```

Comment:
Can we have 2 jobs running in parallel, one for **E2E tests** and the other for **Component tests**?